packages/ui-toolkit/src/components/atoms/ProgressBar/ProgressBar.tsx

```diff
@@ -21,6 +22,7 @@ class ProgressBar extends React.PureComponent<Props> {
     const { isCircular } = this.props;
 
     return (
+      //@ts-ignore
```

Comment:
why did we need to add this ? Also have we added react-waypoint in our package.json for ui-toolkit ?

Reply:
removed ts-ignore. react-waypoint is already there in package.json